### PR TITLE
Update Atlassian documentation link

### PR DESCRIPTION
**Pull request by glasnt**
As per discussion in https://github.com/github/github-services/issues/1145#issuecomment-209806132

### CODE REVIEW

**Comment by glasnt** [comment]
ping @mikemcquaid: CLA signed, and travis green

File: docs/hipchat [hunk 1]
```diff
@@ -1,6 +1,5 @@
-Note: HipChat has a new and improved integration with GitHub. While this
-service will continue to work, you really should check out the new integration.
-Read more here: http://help.hipchat.com/knowledgebase/articles/64389-github-integration
+Note: Atlassian provides it's own native integration for GitHub.
```
Comment (gjtorikian) [inline docs/hipchat]: It should be its, not it's.

Comment (glasnt) [inline docs/hipchat]: Whoops, my bad. I'll amend the commit.

File: docs/hipchat [hunk 2]
```diff
@@ -1,6 +1,5 @@
-Note: HipChat has a new and improved integration with GitHub. While this
-service will continue to work, you really should check out the new integration.
-Read more here: http://help.hipchat.com/knowledgebase/articles/64389-github-integration
+Note: Atlassian provides its own native integration for GitHub.
+Read more here: https://hipchat.com/addons/github-for-hipchat
```
Comment (davidcelis) [inline docs/hipchat]: Could we keep the text noting that we prefer the use of Atlassian's integration and not the hipchat service provided here?

Comment (glasnt) [inline docs/hipchat]: Commit amended.

**Comment by mikemcquaid** [comment]
@glasnt Thanks for your contribution! You rock!

**Pull request merged**